src/lib.rs

```diff
@@ -49,6 +50,13 @@ fn run(args: Args, functions: Vec<(String, RustFunction)>) -> Result<(), Vec<Err
             if let Err(e) = vm.run() {
                 Err(vec![e])
             } else {
+                let mut log = std::fs::File::create("perf.csv").unwrap();
+                for (i, timer) in vm.times_per_op.iter().enumerate() {
+                    if let Some(name) = name_from_index(i) {
+                        let output = format!("'{}'| {}| {}|\n", name, timer.calls, timer.time);
```

Comment:
You can pad with spaces here (`"{<22}|{>8}|{>12}"`) for prettiness. Also, `|` in a csv-file?

Reply:
CSV allows any symbol to be used as a seprator. And we have commas in the Op::* names, since I was too lazy to remove them. So I picked a random character that I know didn't come up.

I like the padding idea though!